mhs/mhs/common/workflow/sync_async.py

```diff
@@ -33,18 +33,24 @@ def __init__(self, config_manager: configuration_manager.ConfigurationManager,
         self.transmission = transmission
         self.party_id = party_id
 
-    def prepare_message(self, interaction_name: str, content: str) -> typing.Tuple[bool, str, str]:
+    def prepare_message(self, interaction_name: str, content: str, message_id: typing.Optional[str] = None) \
+            -> typing.Tuple[bool, str, str]:
```

Comment:
As I was saying earlier google suggests importing modules not classes, however there is an exemption for the typing  [module](http://google.github.io/styleguide/pyguide.html#22-imports)